app/assets/scss/application.scss

```diff
@@ -158,6 +158,10 @@ $path: "/static/images/";
   }
 }
 
+.break-link {
+  word-break: break-all;
```

Comment:
I went for break all as some of the links were absolutely horrible and didn't really have words in them, so breaking everything worked better here.